compilation warnings in ofPixels.cpp : std::move prevents copy elision

on MSYS2, the following warning (repeated for several other lines)  appear : 
```
N:/openframeworks/libs/openFrameworks/graphics/ofPixels.cpp:947:24: warning: moving a local object in a return statement prevents copy elision [-Wpessimizing-move]
  947 |  return std::move(plane);
      |                        ^
N:/openframeworks/libs/openFrameworks/graphics/ofPixels.cpp:947:24: note: remove 'std::move' call
```

--- a/libs/openFrameworks/graphics/ofPixels.cpp
+++ b/libs/openFrameworks/graphics/ofPixels.cpp
@@ -944,7 +944,7 @@ ofPixels_<PixelType> ofPixels_<PixelType>::getPlane(size_t planeIdx){
 		case OF_PIXELS_UNKNOWN:
 			break;
 	}
-	return std::move(plane);
+	return plane;
 }
 
 template<typename PixelType>
@@ -1009,7 +1009,7 @@ ofPixels_<PixelType> ofPixels_<PixelType>::getChannel(size_t channel) const{
 	for(auto p: getConstPixelsIter()){
 		*channelPixel++ = p[channel];
 	}
-	return std::move(channelPixels);
+	return channelPixels;
 }
 
 template<typename PixelType>

--- a/libs/openFrameworks/graphics/ofPixels.h
+++ b/libs/openFrameworks/graphics/ofPixels.h
@@ -1093,7 +1093,7 @@ template<typename PixelType>
 inline ofPixels_<PixelType> ofPixels_<PixelType>::Line::asPixels(){
 	ofPixels_<PixelType> pixels;
 	pixels.setFromExternalPixels(_begin,stride/componentsPerPixel,1,pixelFormat);
-	return std::move(pixels);
+	return pixels;
 }
 
 //----------------------------------------------------------------------
@@ -1101,7 +1101,7 @@ template<typename PixelType>
 inline const ofPixels_<PixelType> ofPixels_<PixelType>::Line::asPixels() const{
 	ofPixels_<PixelType> pixels;
 	pixels.setFromExternalPixels(_begin,stride/componentsPerPixel,1,pixelFormat);
-	return std::move(pixels);
+	return pixels;
 }
 
 //----------------------------------------------------------------------

